From 855a1f88363406a2f1948a88ebe02056a8f22d35 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Tue, 10 Aug 2010 13:37:00 +0000
Subject: [PATCH] commit: fix test broken by jn/commit-no-change-wo-status
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

"commit: suppress status summary when no changes staged" in Jonathan
Nieder's jn/commit-no-change-wo-status series changed git commit
--dry-run output. But didn't update the t/t6040-tracking-info.sh test,
which relied on the old output.

Change the test to check the new output, and additionally test stderr
there while I'm at it.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---

This is 1/2 things causing black smoke in pu right now.

 t/t6040-tracking-info.sh |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/t/t6040-tracking-info.sh b/t/t6040-tracking-info.sh
index 1785e17..5f50f70 100755
--- a/t/t6040-tracking-info.sh
+++ b/t/t6040-tracking-info.sh
@@ -70,8 +70,9 @@ test_expect_success 'status' '
 		git checkout b1 >/dev/null &&
 		# reports nothing to commit
 		test_must_fail git commit --dry-run
-	) >actual &&
-	grep "have 1 and 1 different" actual
+	) >actual 2>actual-err &&
+	grep "nothing to commit" actual &&
+	grep "Already on" actual-err
 '
 
 test_expect_success 'status when tracking lightweight tags' '
-- 
1.7.1

